test/error_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+describe Lotus::Model::Error do
+  it 'inherits from ::StandardError' do
+    Lotus::Model::Error.superclass.must_be StandardError
+  end
+
+  it 'parent to all custom exception' do
```

Comment:
I don't think this test adds more value because you have already tested it in `test/integration/repository_test.rb`